src/style.css

```
@@ -1,3 +1,79 @@
+html {
+  --color-col: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  color: var(--color-col);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  position: relative;
+}
+
+.nav__item {
+  margin-right: 20px;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
```

you did something wrong if you need to set it like fractional values
```suggestion
  padding: 22.5px 0
```